hieradata/integration.yaml

```diff
@@ -23,8 +23,6 @@ govuk::apps::publicapi::backdrop_host: 'www.preview.performance.service.gov.uk'
 govuk::apps::rummager::aws_s3_bucket_name: govuk-api-elasticsearch-snapshots-integration
 
 govuk::apps::smartanswers::expose_govspeak: true
-govuk::apps::dfid_transition::enabled: true
-govuk::apps::dfid_transition::enable_procfile_worker: true
```

Comment:
dfid_transition will still be enabled in integration. I've set these to `true` in common.yml which this environment inherits from.